src/Components/TableBody/TableBody.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Person } from '../../People';
+
+type Props = {
+  people: Person[],
+};
+
+const PersonRow: React.FC<Props> = ({ people }) => (
+  <>
+    {people.map(person => {
+      const {
+        slug,
+        name,
+        sex,
+        born,
+        died,
+        motherName,
+        fatherName,
+      } = person;
```

Comment:
`PersonRow` should be unique for each person, move this map outside of this component and pass just 1 person info to it